### PR TITLE
Internal Affairs Agents

### PR DESCRIPTION
**Pull request by ike709**
:cl: ike709, HUD sprite from Paradise
add: Internal Affairs Agents are now a thing.
del: Lawyers have been removed and replaced by IAAs.
/:cl:

~~They don't have an office mapped in yet, so currently they just spawn wherever. Someone suggested replacing the showroom with the IAA office, and if we want to keep the showroom it could be recreated in the abandoned office near arrivals.~~ 

Law office is IAA office now.

### CODE REVIEW

File: code/game/machinery/computer/card.dm [hunk 1]
```diff
@@ -37,7 +37,8 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		"Research Director",
 		"Chief Medical Officer",
 		"Blueshield",
-		"Brig Physician")
+		"Brig Physician",
+		"Internal Affairs Agent")
```
Comment (Birdtalon) [inline code/game/machinery/computer/card.dm]: I don't think it should be personally.

**Comment by FlattestGuitar** [comment]
While I'm not sure what the plan here is exactly, I'd say we should rip out lawyers while adding IAAs.

**Comment by ike709** [comment]
I've seen far greater support in discord for keeping lawyers alongside IAAs.

Also the plan is that they yell at people for not following SOP and sometimes report people to CentCom.

**Comment by Qustinnus** [comment]
you cant even fix printers nerd @ike709

**Comment by ike709** [comment]
@Qustinnus what are you on about

**Comment by Qustinnus** [comment]
on ftl you couldnt fix the printers

**Comment by ike709** [comment]
Actually I couldn't make a TGUI for them because they used NanoUI on other codebases. Funce handled that so I took care of the rest.

**Comment by FlattestGuitar** [comment]
Following a talk with the staff team, we don't want to have *both* IAAs and Lawyers. For now, we just want to switch the name of the Lawyers' job to this and adding basic departmental access, with stuff like ticketing machines and SOP books and stuff added later down the road. 

I won't close the PR if you want to change it to that.

Apologies for the late response.

**Comment by ike709** [comment]
I'll work on it.

**Comment by ike709** [comment]
Lawyers have been more or less entirely removed from existence. Remaining stuff that I either won't change or doesn't need to be changed:

Map-wise, only Metastation has been entirely changed.
No icons have been renamed. Fuck that.
ACCESS_LAWYER still exists so that I don't need to go in and alter the maps even more.

**Comment by ike709** [comment]
aaaaaaaah conflicts

**Comment by ike709** [comment]
conflicts resolved

**Comment by ike709** [comment]
Reminder that this is ready.

**Comment by ike709** [comment]
Daily remind that this is ready.

**Comment by Qustinnus** [comment]
daily reminder to stay hydrated

**Comment by ike709** [comment]
Daily reminder that this is ready.

File: code/__DEFINES/jobs.dm [hunk 2]
```diff
@@ -37,9 +37,9 @@
...
-#define MIME			(1<<12)
-#define ASSISTANT		(1<<13)
-#define BLUESHIELD		(1<<14)
\ No newline at end of file
+#define CHAPLAIN		(1<<9)
+#define CLOWN			(1<<10)
+#define MIME			(1<<11)
+#define ASSISTANT		(1<<12)
+#define BLUESHIELD		(1<<13)
+#define IAA		(1<<14)
```
Comment (FlattestGuitar) [inline code/__DEFINES/jobs.dm]: This define will need to go where `LAWYER` was before, otherwise we might screw something up.

**Comment by ike709** [comment]
The hell happened to travis?

**Pull request merged**